### PR TITLE
Create re-usable Message Screen template

### PR DESCRIPTION
**Pull request by mateuszpmroz**
image

### CODE REVIEW

File: src/screens/MessageScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { useNavigationParam } from 'react-navigation-hooks';
+import { Button } from 'components/Button';
+
+import { typography, palette } from 'styles';
+
+export const MessageScreen = () => {
+  const title: string = useNavigationParam('title');
+  const source: FastImageSource = useNavigationParam('source');
+  const description: string = useNavigationParam('description');
+  const button: ButtonProps | undefined = useNavigationParam('button');
```
Comment (mateuszpmroz) [inline src/screens/MessageScreen.tsx:14]: Maybe you know if there's a possibility to make these params required?

Example of usage:
```
  NavigationService.navigate('Message', {
            title: 'Success',
            description: 'Your contatct has been successfully deleted. You can now return to Address book.',
            source: images.success,
            button: { title: 'Return to Address book' }
          })
```
In that case we can omit the param and we don't have an error :thinking:

File: src/screens/MessageScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import { Text, View, StyleSheet } from 'react-native';
+import { useNavigationParam } from 'react-navigation-hooks';
+import { Button } from 'components/Button';
+
+import { typography, palette } from 'styles';
+
+export const MessageScreen = () => {
+  const title: string = useNavigationParam('title');
+  const source: FastImageSource = useNavigationParam('source');
+  const description: string = useNavigationParam('description');
```
Comment (mateuszpmroz) [inline src/screens/MessageScreen.tsx:13]: Maybe you know if there's a possibility to make these params required?

Example of usage:
```
  NavigationService.navigate('Message', {
            title: 'Success',
            description: 'Your contatct has been successfully deleted. You can now return to Address book.',
            source: images.success,
            buttonProps: { title: 'Return to Address book' }
          })
```
In that case we can omit the param and we don't have an error :thinking:

Comment (mateuszpmroz) [reply src/screens/MessageScreen.tsx:20]: Actually I've updated the `NavigationService` and now we have suggestions for that screen.

File: src/screens/MessageScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import { typography, palette } from 'styles';
+
+export interface MessageProps {
+  title: string;
+  source: FastImageSource;
+  description: string;
+  buttonProps?: ButtonProps;
+}
+
+export const MessageScreen = () => {
```
Comment (bklukaczewski) [inline src/screens/MessageScreen.tsx:17]: I was thinking that you would create just `MessageScreenTemplate` and then in the actual screen, like `AddCustomerSucessScreen` you would use it.

However your approach is also good 👍 even less code. Slightly less verbose. A bit worse for analytics ( All message screen are with the same name for analytics) but there's no one correct way.

**Pull request merged**